### PR TITLE
Add two pane view to calendar panel

### PR DESCRIPTION
**Pull request by bramkragten**
## Proposed change

Intoduces a two pane view, that turns into a 1 pane view on narrow screens:

CleanShot 2023-10-19 at 12 14 26

On mobile the header becomes a drop down menu that allows to pick the calendars that you want to show:

CleanShot 2023-10-19 at 12 14 48

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/components/ha-button-menu.ts [hunk 1]
```diff
@@ -26,6 +26,8 @@ export class HaButtonMenu extends LitElement {
 
   @property({ type: Boolean }) public fixed = false;
 
+  @property({ type: Boolean }) public noAnchor = false;
```
Comment (piitaya) [inline src/components/ha-button-menu.ts:29]:
```suggestion
  @property({ type: Boolean, attribute: "no-anchor" }) public noAnchor = false;
```

**Review by piitaya — COMMENTED** [review]
I noticed 2 things.

The lines are not aligned in some screen sizes. Also, do we really need this line in the top bar? 😅

CleanShot 2023-10-23 at 11 48 52


On mobile, does the dropdown needs to be closed on each checkbox click?

https://github.com/home-assistant/frontend/assets/5878303/21874333-2962-40b2-b64e-eabef56c5045

**Comment by bramkragten** [comment]
> Also, do we really need this line in the top bar?

Yes, I think we do, especially if we are going to move controls to the right side of the top bar

**Review by piitaya — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -52,10 +52,12 @@
     "@lezer/highlight": "1.1.6",
     "@lit-labs/context": "0.4.1",
     "@lit-labs/motion": "1.0.4",
+    "@lit-labs/observers": "2.0.1",
```
Comment (steverep) [inline package.json:55]: FYI, this introduced a duplicate of `@lit/reactive-element` (bumping it to the next major version).

Comment (steverep) [reply package.json:55]: Adjusted in #18484

**Pull request merged**